tools/alpine/Makefile

```diff
@@ -2,22 +2,38 @@
 
 ORG?=linuxkit
 IMAGE=alpine
+
+PKG_DEPS=packages
+ifeq ($(shell uname -m), x86_64)
 BASE=alpine:3.6
+PKG_DEPS += packages.x86_64
+endif
+
+ifeq ($(shell uname -m), aarch64)
+BASE=arm64v8/alpine:3.6
+PKG_DEPS += packages.aarch64
+endif
+
+ifeq ($(DOCKER_CONTENT_TRUST),)
+ifndef NOTRUST
+export DOCKER_CONTENT_TRUST=0
+endif
+endif
 
 default: push
 
 show-tag:
 	@sed -n -e '1s/# \(.*\/.*:[0-9a-f]\{40\}\)/\1/p;q' versions
 
-hash: Dockerfile Makefile packages
-	DOCKER_CONTENT_TRUST=1 docker pull $(BASE)
-	docker build --no-cache -t $(IMAGE):build .
+hash: Dockerfile Makefile $(PKG_DEPS)
+	docker pull $(BASE)
```

Comment:
@justincormack @riyazdf  
Ah, thanks for the remind! I totally agree with that we need to keep x86_64 trust checking intact, I am missing that. So now I made below changes:
`diff --git a/tools/alpine/Makefile b/tools/alpine/Makefile
index 9d9b184..4fcad2a 100644
--- a/tools/alpine/Makefile
+++ b/tools/alpine/Makefile
@@ -3,6 +3,7 @@
 ORG?=linuxkit
 IMAGE=alpine
 
+DOCKER_CONTENT_VAR=1
 PKG_DEPS=packages
 ifeq ($(shell uname -m), x86_64)
 BASE=alpine:3.6
@@ -12,12 +13,7 @@ endif
 ifeq ($(shell uname -m), aarch64)
 BASE=arm64v8/alpine:3.6
 PKG_DEPS += packages.aarch64
-endif
-
-ifeq ($(DOCKER_CONTENT_TRUST),)
-ifndef NOTRUST
-export DOCKER_CONTENT_TRUST=0
-endif
+DOCKER_CONTENT_VAR=0
 endif
 
 default: push
@@ -26,14 +22,14 @@ show-tag:
        @sed -n -e '1s/# \(.*\/.*:[0-9a-f]\{40\}\)/\1/p;q' versions
 
 hash: Dockerfile Makefile $(PKG_DEPS)
-       docker pull $(BASE)
+       DOCKER_CONTENT_TRUST=$(DOCKER_CONTENT_VAR) docker pull $(BASE)
        docker build --no-cache --build-arg BASE=$(BASE) -t $(IMAGE):build .
        docker run --rm $(IMAGE):build sh -c 'echo Dockerfile /lib/apk/db/installed $$(find /mirror -name '*.apk' -type f) $$(find /go/bin -type f) | xargs cat | sha1sum' | sed 's/ .*//' > $@
 
 push: hash
-       docker pull $(ORG)/$(IMAGE):$(shell cat hash) || \
+       DOCKER_CONTENT_TRUST=$(DOCKER_CONTENT_VAR) docker pull $(ORG)/$(IMAGE):$(shell cat hash) || \
                (docker tag $(IMAGE):build $(ORG)/$(IMAGE):$(shell cat hash) && \
-                docker push $(ORG)/$(IMAGE):$(shell cat hash))
+                DOCKER_CONTENT_TRUST=$(DOCKER_CONTENT_VAR) docker push $(ORG)/$(IMAGE):$(shell cat hash))`

With this change, we can keep x86_64 has the same behavior as it does before the update.
Let me know if any suggestions. Thanks for the review!